metadata/index.html

```diff
@@ -880,6 +852,10 @@
         <p>
           The description MAY contain <a title="inherited property">inherited properties</a> to describe cells within tables that use this schema.
         </p>
+        <section>
+          <h4>Schema Compatibility</h4>
+          <p>Two Schemas are <dfn title="schema compatible">compatible</dfn> if they have the same number of non-<a href="#column-virtual"><code>virtual</code></a> columns, and all non-<a href="#column-virtual"><code>virtual</code></a> columns are compatible with each other. Columns at the same index are compatible if there is a non-empty case-insensitive intersection between the percent-decoded <a href="#column-name"><code>name</code></a> and <a href="#column-titles"><code>titles</code></a> values.</p>
+        </section>
```

Comment:
We do need to add a case when there is an empty column description, as would be created in embedded metadata if `headers` is false. In this case, there are no titles, but we have a column count, and these columns should match any column. See, for example test019 in http://w3c.github.io/csvw/tests/.